modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/utils/APIService.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {fetch} from 'frontend-js-web';
+
+export default {
+	getLayoutReportsIssues(layoutReportsIssuesURL, namespace) {
+		return _fetchWithError(layoutReportsIssuesURL, {
+			body: _getFormDataRequest({}, namespace),
+			method: 'POST',
+		});
+	},
+};
+
+/**
+ *
+ *
+ * @export
+ * @param {Object} body
+ * @param {string} prefix
+ * @param {FormData} [formData=new FormData()]
+ * @returns {FormData}
+ */
+export function _getFormDataRequest(body, prefix, formData = new FormData()) {
+	Object.entries(body).forEach(([key, value]) => {
+		formData.append(`${prefix}${key}`, value);
+	});
+
+	return formData;
+}
+
+/**
+ * Wrapper to `fetch` function throwing an error when `error` is present in the response
+ */
+function _fetchWithError(url, options = {}) {
```

Comment:
Seems that:

- We don't need to export `_getFormDataRequest` (it is only used internally)
- We should never export a function with a leading underscore anyway
- We don't need to mark these functions as "private" by using an underscore

ie. I think we can delete the underscores, and get rid of the `export`

Again, because you have a clean CI run, feel free to forward if you want and just fold these changes in to the next time you touch this file.

Reply:
Ok! Sure! I'll update it and push force 👍